api.toml

```diff
@@ -36,6 +36,9 @@
         # /address/:address/username will return the username of a given account
         { Name = "/:address/username", Open = true },
 
+        # /address/:address/keys will return all the key-value pairs of a given account
+        { Name = "/:address/keys", Open = true },
```

Comment:
maybe there's a small inconsistency here with the /keys while the next one is /key/:key but it's for another PR

Reply:
@bogdan-rosianu ?